target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -45,6 +45,13 @@ nbg6817)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "$board:amber:wifi5" "phy0radio"
 	ucidef_set_led_netdev "wan" "WAN" "$board:white:internet" "eth1"
 	;;
+vr2600v)
+	ucidef_set_led_usbdev "phone" "Phone" "${board}:white:phone" "1"
```

Comment:
Meh. ```power: phone {}```. Missed that. Please don't do that. Use the alias phone.

Nevertheless, the ucidef_set_led_usbdev line can be dropped. It is wrong on multiple levels. I mean usbdev as status?